extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/supports/blocking/BlockingServerInterceptor.java

```diff
@@ -62,13 +64,20 @@ public Boolean apply(String name) {
         boolean isBlocking = cache.computeIfAbsent(fullMethodName, this);
 
         if (isBlocking) {
-            ReplayListener<ReqT> replay = new ReplayListener<>();
-
+            final ManagedContext requestContext = getRequestContext();
+            ContextState state = requestContext.getState();
```

Comment:
Hm, this may throw a `ContextNotActiveException`... Is it handled somewhere?

Reply:
Hmm, it was my understanding that when this is called, you will always have a req. context active.

Reply:
No, the state should always be there... OTOH, if it isn't, it may not be a critical error, maybe I can just log a warning and activate a new context, WDYT?

Reply:
Maybe just document that assumption?

My understanding is that if `BlockingServerInterceptor` is present in the interceptor chain, it is always preceded by `GrpcRequestContextGrpcInterceptor`, and so request context should always be active.

Reply:
I'll add a comment then

Reply:
done